keyboards/handwired/prime_exl/prime_exl.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2018 Holten Campbell
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef PRIME_EXL_H
+#define PRIME_EXL_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
The top half of this macro, as well as the layer definitions in your `keymap.c` should roughly resemble the physical layout of the keyboard, so people who build this keyboard can tell at a glance which keys to edit to change their keymap.

Reply:
As @noroadsleft has said.  

The bottom half is how the matrix is physically layout out.  But the top half can be reordered any way that you like, so that it is easy to read by humans. 

A good example of this is the Planck Rev6. 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/planck.h#L53-L67

Reply:
@drashna should be good to go now.